src/Common/labelHeader.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import { Icon } from 'react-native-elements';
+import PropTypes from 'prop-types';
+import styles from './themeStyle';
+import themeColor from './Color';
+
+export default function LabelHeader(props) {
+  const {
+    icon, name, type, amount,
+  } = props;
+
+  return (
+    name
+      ? (
+        <LinearGradient
+          colors={type === 'out' ? themeColor.gradientColor.red : themeColor.gradientColor.green}
+          start={[0.1, 0.9]}
+          end={[0.9, 0.1]}
+        >
+          <View style={[styles.headerFormat, { height: 140, justifyContent: 'space-between' }]}>
+            <View style={{ alignItems: 'flex-start' }}>
+              <Icon name={icon} color="#ffffff" size={40} />
+              <Text style={{ color: '#ffffff', fontSize: 20, fontWeight: '400' }}>{name || 'undefined'}</Text>
```

Comment:
may create a style object for this, to make the jsx file much more simple

Reply:
done